src/MusicStore/Program.cs

```diff
@@ -22,6 +22,7 @@ public static void Main(string[] args)
                 .UseContentRoot(Directory.GetCurrentDirectory())
                 .UseConfiguration(config)
                 .UseIISIntegration()
+                .UseUrls("http://127.0.0.1:5000")
```

Comment:
Rather than hardcoding what you want here, I'd suggest passing this in at the command line. https://docs.microsoft.com/en-us/aspnet/core/fundamentals/servers/kestrel?view=aspnetcore-2.2#endpoint-configuration

Line 17 should change to pass in the command line args.

Then down below you can retrieve the listen URL from `host.Configuration`.

Reply:
There could be the issue of passing `--urls 0.0.0.0:5000` for example which is valid to bind to, but not connect to. Since we already hardcoded `localhost:5000` at https://github.com/aspnet/JitBench/pull/99/files#diff-eed82aa1208a1c635d68d21340f5ffc8L39, it would make sense IMO to hardcode it here as well, especially since `127.0.0.1` is a universally recognised IP which does not introduce any other dependency.